src/main/java/controller/RacingGameController.java

```diff
@@ -0,0 +1,37 @@
+package controller;
+
+import java.util.List;
+import model.Cars;
+import service.RacingGameService;
+import splitter.CarNameSplitter;
+import view.InputView;
+import view.OutputView;
+
+public class RacingGameController {
+    private final InputView inputView;
+    private final OutputView outputView;
+    private final CarNameSplitter carNameSplitter;
+    private final RacingGameService racingGameService;
+
+    public RacingGameController(InputView inputView, OutputView outputView, CarNameSplitter carNameSplitter,
+                                RacingGameService racingGameService) {
+        this.inputView = inputView;
+        this.outputView = outputView;
+        this.carNameSplitter = carNameSplitter;
+        this.racingGameService = racingGameService;
+    }
+
+    public void run() {
+        String inputNames = inputView.getCarName();
+        List<String> carNames = carNameSplitter.splitCarNames(inputNames);
+
+        Cars cars = new Cars(carNames);
```

Comment:
이 부분은 결국 Cars를 만드는 부분으로 보여집니다.
```suggestion
        Cars cars = initCars();
        
        private Cars initCars(){
        String inputName = inputView.getCarName();
        List<String> carNames = carNameSplitter.splitCarNames(inputNames);
        return new Cars(carNames)
        }
```
요런식으로 메서드로 뽑아서 가독성 좋게 구현하면 어떨까 라고 생각해봤습니다. 어떻게 생각하시나요?

Reply:
한 번 더 포장해주는거 너무 좋은 것 같습니다! 피드백 수용해서 다음주에 적용시켜보도록 할게요 !!